smdebug/tensorflow/collection.py

```diff
@@ -73,8 +73,10 @@ def add_for_mode(self, arg, mode=None):
          Adds tensors to the collection from a given Operation, Tensor, Variable or MirroredVariable
          :param arg: the argument to add to collection
          """
+        get_logger().debug(f"type:{type(arg)} arg:{arg} self:{id(self)}")
```

Comment:
Did you intend to remove before PR? If not, you might want to add some description before.